### PR TITLE
Fixes

### LINKED ISSUE #7: Use profile path to get wallet and config files
#### Summary
`wallet.dat` and `config.toml` should be inside the profile path specified (and not inside the `RUSK_PROFILE_PATH`

#### Additional context
N/A

### PR DESCRIPTION
**Pull request by herr-seppia**
### governance-core 
- change Governance constructor to use config file path
- fix `TransferMap` logic
- fix connection status assertion
### governance-cli: 
- use `--profile` to get wallet and config

Resolves #7 #8

### CODE REVIEW

File: governance-core/src/lib.rs [hunk 1]
```diff
@@ -67,9 +69,10 @@ impl Governance {
             .connect_with_status(transport_tcp, |s| info!("Status: {}", s))
             .await?;
 
-        assert!(!wallet.is_online(), "Wallet is not online");
+        assert!(wallet.is_online(), "Wallet is not online");
```
Comment (Daksh14) [inline governance-core/src/lib.rs:72]: Any reason for removing the bang?

File: governance-core/src/json.rs [hunk 2]
```diff
@@ -53,25 +53,22 @@ pub fn json_bytes<T: AsRef<[u8]>>(bytes: T) -> io::Result<TransferMap> {
...
 
                     let mut tx = Transfer::new(size, occurrence);
-                    // set the security of the transfer
-                    map.security = security;
                     match cause {
                         Cause::Rebalance => {
                             if size < 0.0 {
                                 tx.amount(-size);
-                                map.insert_tx(tx.withdraw(from));
+                                map.insert_tx(tx.withdraw(from), security);
```
Comment (Daksh14) [inline governance-core/src/json.rs:60]: Do you think we should set the argument of insert as K, V like the wider ecosystem like `map.insert_tx(security, tx.withdraw(from));`

**Review by Daksh14 — CHANGES_REQUESTED** [review]

File: governance-core/src/json.rs (hunk 2 above)
Comment (herr-seppia) [reply governance-core/src/json.rs:60]: Yes, it will increase readability indeed

File: governance-core/src/lib.rs (hunk 1 above)
Comment (herr-seppia) [reply governance-core/src/lib.rs:72]: Assert ensures that the first argument is true, otherwise it panics.
We need the wallet online

**Review by Daksh14 — APPROVED** [review]

**Pull request merged**